setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="vxvas2nets",
+    version="0.6.0a",
+    url='http://github.com/praekelt/vumi-vas2nets',
+    license='BSD',
+    description="A Vas2Nets USSD transport for Vumi.",
+    long_description=open('README.rst', 'r').read(),
+    author='Praekelt Foundation',
+    author_email='dev@praekeltfoundation.org',
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[
+        'vumi>=0.6.0',
+    ],
```

Comment:
`vumi` requires Twisted with an appropriate version restriction, so there's no need to also require it here.